DATA/SConscript

```diff
@@ -155,14 +155,22 @@ env.Alias('crosstalk', crosstalk)
 
 #PTC
 ptc = env.Command([os.path.join(REPO_ROOT, 'ci_cpp_ptc'),
-                         os.path.join(REPO_ROOT, "calib", 'v00', "ptc")], crosstalk,
-                        [getPipeTaskCmd('ptc', exposureDict['ptcExposurePairs'],
-                                        'measurePhotonTransferCurve.yaml'),
-                         getCertifyCmd('ptc')])
+                   os.path.join(REPO_ROOT, "calib", 'v00', "ptc")], crosstalk,
```

Comment:
Are you just deleting tabs here, is there something more substantial going on?

Reply:
It should just be space change there.  I don't think SConscript files get run through flake8, but this would have failed the "E127 continuation line over-indented for visual indent".  I copied this block to make the BFK one, and when my editor auto-fixed the spacing there, I did the same here.